docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md

```diff
@@ -270,6 +270,31 @@ class EmbeddedPaymentPageRouteProviderPlugin extends AbstractRouteProviderPlugin
 4. In `src/Pyz/Yves/Router/RouterDependencyProvider.php`, add a router plugin to `RouterDependencyProvider::getRouteProvider()`.
 
 
+## Using Stripe in Marketplace context
+
+### Merchant Onboarding Process
+
+To be able to transfer money from the Marketplace Account to the Merchants, Merchants need to be onboarded to the Stripe App. When you configured the Stripe App in Marketplace mode, the Merchant onboarding process is initiated. The process is as follows:
+
+- Merchants need login to the Merchant Portal.
+- Merchants need to open the "Payment Settings" section.
+- On the "Payment Settings" the Merchant will see a new section "Stripe Onboarding".
+- The Merchant needs to click on the "Connect with Stripe" button.
+- An onboarding page of Stripe will be opened and the Merchant needs to fill in his data.
+- After the onboarding is completed, the Merchant will be redirected back to the Merchant Portal.
```

Comment:
What onboarding status does a merchant get after this step?

Reply:
They will be in state incomplete until Stripe sends a new state. Incomplete can also be there when the onboarding process is aborted in the middle but will be continued at the place where the Merchant left off.